app/models/ncr/work_order.rb

```diff
@@ -47,6 +47,11 @@ def self.purchase_amount_column_name
       message: "must be three letters or numbers"
     }, allow_blank: true
 
+    scope :for_fiscal_year, lambda { |year|
```

Comment:
since the logic here and in `Proposal` is exactly the same, do you think it makes sense for us to include it elsewhere? 

I am not opposed to duplicated code when it makes sense, but the reason I first identified this was that I was looking for code in NCR::WorkOrder that was not specific to Work Orders. 

Seems like fiscal year logic might be helpful for reporting for other clients. In that case, we'd want this scope on those client data classes as well. Might be wrong to include this in the FiscalYear mixin, but it does seem like it should be outside of Proposal and WorkOrder. Does that make sense?